Bottom: 2b485f17d72d951a50a5ec93c40edbddc121fa0b
Top:    5aaba4fd36cba9e062357c3423a56a217928d394
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-11 16:56:00 -0500

Add a new vagrant machine for the builder


---
diff --git a/ansible-playbook/host_vars/buildbot-worker.novatech-llc.com/privileges.yml b/ansible-playbook/host_vars/buildbot-worker.novatech-llc.com/privileges.yml
new file mode 100644
index 00000000..fa188ba9
--- /dev/null
+++ b/ansible-playbook/host_vars/buildbot-worker.novatech-llc.com/privileges.yml
@@ -0,0 +1,3 @@
+---
+
+ansible_user: vagrant
